Incorrect guidance from test resourceSanitizer

When using the Deno.Command() api, if one fails to manually close the streams to prevent resource leaks, Deno test sanitizer wonderfully suggests a fix  like so:

```
error: AssertionError: Test case is leaking 1 resource:

 - A child process stderr (rid 19) was opened during the test, but not closed during the test. Close the child process stderr by calling `proc.stderr.close()`.

    at assert (deno:ext/web/00_infra.js:295:13)
    at resourceSanitizer (deno:runtime/js/40_testing.js:413:7)
    at async Object.exitSanitizer [as fn] (deno:runtime/js/40_testing.js:431:9)
    at async runTest (deno:runtime/js/40_testing.js:836:7)
    at async runTests (deno:runtime/js/40_testing.js:1091:22)
```

However, it took me a while to realize why the suggested solution doesn't work. It's because, while stdin needs to be closed with `stdin.close()`, stderr needs to be closed with `stderr.cancel()` . I believe the error message should be updated to reflect that. Small issue, but the incorrect advice can cause a great deal of confusion.

--- a/cli/js/40_testing.js
+++ b/cli/js/40_testing.js
@@ -314,7 +314,7 @@
       case "httpConn":
         return "Close the inbound HTTP connection by calling `httpConn.close()`.";
       case "httpStream":
-        return "Close the inbound HTTP request by responding with `e.respondWith().` or closing the HTTP connection.";
+        return "Close the inbound HTTP request by responding with `e.respondWith()` or closing the HTTP connection.";
       case "tcpStream":
         return "Close the TCP connection by calling `tcpConn.close()`.";
       case "unixStream":
@@ -344,9 +344,9 @@
       case "childStdin":
         return "Close the child process stdin by calling `proc.stdin.close()`.";
       case "childStdout":
-        return "Close the child process stdout by calling `proc.stdout.close()`.";
+        return "Close the child process stdout by calling `proc.stdout.close()` or `await child.stdout.cancel()`.";
       case "childStderr":
-        return "Close the child process stderr by calling `proc.stderr.close()`.";
+        return "Close the child process stderr by calling `proc.stderr.close()` or `await child.stderr.cancel()`.";
       case "child":
         return "Close the child process by calling `proc.kill()` or `proc.close()`.";
       case "signal":

